examples/ConvertPyImageJDataset.ipynb

```diff
@@ -0,0 +1,1153 @@
+{
+ "cells": [
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "cffe4de0",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "import sys, os\n",
+    "!conda install --yes --prefix {sys.prefix} -c conda-forge openjdk=8\n",
```

Comment:
I wonder if you need to use conda at all, since you use `setup-java` to install openjdk and maven anyway, which are the non-Python dependencies of pyimagej. You could probably just skip this step, and set JAVA_HOME correctly to wherever setup-java puts it, no?

Reply:
I guess you need it for Binder, though? It's fine then.

Reply:
Yes, good idea @ctrueden ! I tried conda, then moved to what you have set up in CI for pyimagej -- I will try first removing conda then increment on the other debugging steps you suggested.